gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-rc-2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
```

Comment:
Don't you have to regenerate the gradle wrapper files by executing ./gradlew wrapper if you want this change to be valid?

Reply:
Not sure if Intellj did something, but everything works, I have executed `./gradlew wrapper` and upload the new version of the wrapper